docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -6,6 +6,9 @@
   - [Zowe API ML services](#zowe-api-ml-services)
   - [Zowe API ML TLS requirements](#zowe-api-ml-tls-requirements)
   - [Authentication for API ML services](#authentication-for-api-ml-services)
+    - [Authentication endpoints](#authentication-endpoints)
+    - [z/OSMF Authentication Provider](#z/osmf-authentication-provider)
```

Comment:
This link does not work

Reply:
I've just tested. They worked for me.

Reply:
It does not work on Github. I think slash is causing it ...

https://github.com/zowe/docs-site/blob/5e27df1b2d6a1c9c5545469b782ee4097f607ab2/docs/extend/extend-apiml/api-mediation-security.md#z/osmf-authentication-provider

Reply:
Yes it's not working, probably is the slash as Jirka said. I will try to fix it

Reply:
Nothing, all the title using `/` are broken, I've no idea how to fix them without unifying maybe the word "z/OSMF" into "zOSMF"

Reply:
It seems to work fine on

https://zowe.github.io/docs-site/latest/extend/extend-apiml/api-mediation-security.html#trust-a-z-osmf-certificate

Sorry :-)